### ISSUE TITLE
Fonts are gray/dimmed

### ISSUE THREAD

**Issue by SamSoldatenko** [microsoft/vscode#175121]
Type: <b>Bug</b>

I think it is design BUG.
Why all the fonts are gray. Look at screenshoot. EVERYTHING IS GRAY!
OMG!!!

I'm 47. I have to use glasses. Gray fonts DESTROYS my eyes!

What is the problem with PLAIN OLD BLACK color?!

<img width="1230" alt="Screenshot 2023-02-22 at 14 50 21" src="https://user-images.githubusercontent.com/10289657/220644032-3cb5e37d-371a-4a43-afe7-776a7c48f40b.png">

VS Code version: Code 1.75.1 (Universal) (441438abd1ac652551dbe4d408dfcec8a499b8bf, 2023-02-08T21:34:59.000Z)
OS version: Darwin x64 22.3.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|2, 1, 2|
|Memory (System)|32.00GB (2.20GB free)|
|Process Argv|--crash-reporter-id a4fea2df-943d-4496-80a7-30335cd53dd6|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (16)</summary>

Extension|Author (truncated)|Version
---|---|---
isort|ms-|2022.8.0
python|ms-|2023.2.0
vscode-pylance|ms-|2023.2.30
jupyter|ms-|2023.1.2010391206
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.14
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5
java|red|1.15.0
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-java-debug|vsc|0.48.0
vscode-java-dependency|vsc|0.21.1
vscode-java-pack|vsc|0.25.7
vscode-java-test|vsc|0.37.1
vscode-maven|vsc|0.40.4

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaac:30438847
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
vsccc:30610678
pyindex848:30662994
nodejswelcome1:30587005
3biah626:30602489
pyind779cf:30662993
f6dab269:30613381
pythonsymbol12cf:30657549
2i9eh265:30646982
vscodeenable:30660116

```

</details>

**Comment by colin-grant-work** [comment]
@SamSoldatenko, if you prefer very-dark on very-light you may want to try a high-contrast theme. F1 > Preferences: Color Theme > Light High Contrast.

**Comment by SamSoldatenko** [comment]
No. High-contrast theme makes other, non-text elements too distracting and ugly.
I need **normal** contrast theme. Just don't make text to be "low contrast"

**Comment by vince-fugnitto** [comment]
@SamSoldatenko if a given theme does not suit your needs you can always:
- update `workbench.colorCustomizations` in your preferences to customise the look of your application
- use a different theme, or perhaps create your own which has better contrast

**Comment by colin-grant-work** [comment]
To expand on @vince-fugnitto's suggestion, these color customizations would turn all of the items you pointed to in your screenshot black:

```json
    "workbench.colorCustomizations": {
        "foreground": "#000000",
        "descriptionForeground": "#000000",
        "tab.inactiveForeground": "#000000"
    }
```
